### PR TITLE
Get attached VM name for attached volume in admin CLI

### PR DESCRIPTION
**Pull request by govint**
Modified to get the VM name for attached disks from VM UUID. 

...
snap-disk1        bigone     Ubuntu-16-04                    Wed Nov  9 12:43:47 2016  detached        N/A     100.00MB  17.00MB   ext4             read-write  persistent
clonedisk         bigone     photon-custom-hw10-1.0-13c08b6  Fri Oct 28 07:12:04 2016  detached        N/A     100.00MB  100.00MB  ext4             read-write  independent_persistent
try-attach        bigone     1                               Thu Nov 17 05:10:33 2016  detached        N/A     100.00MB  15.00MB   ext4             read-write  independent_persistent

docker run --rm -it -w /data -v try-attach:/data busybox
/data # 

...
snap-disk1        bigone     Ubuntu-16-04                    Wed Nov  9 12:43:47 2016  detached        N/A     100.00MB  17.00MB   ext4             read-write  persistent
clonedisk         bigone     photon-custom-hw10-1.0-13c08b6  Fri Oct 28 07:12:04 2016  detached        N/A     100.00MB  100.00MB  ext4             read-write  independent_persistent
try-attach        bigone     1                               Thu Nov 17 05:10:33 2016  1               N/A     100.00MB  15.00MB   ext4             read-write  independent_persistent

### CODE REVIEW

File: esx_service/vmdk_ops.py [hunk 1]
```diff
@@ -468,6 +468,10 @@ def findVmByUuid(vm_uuid):
     vm = si.content.searchIndex.FindByUuid(None, vm_uuid, True, False)
     return vm
 
+def vm_uuid2name(vm_uuid):
+    vm = findVmByUuid(vm_uuid)
+    if vm:
+        return vm.config.name
```
Comment (msterin) [inline esx_service/vmdk_ops.py]: it does not return anything if `not vm`. It's OK in python (will just return None) but it looks unpleasant. 
Something like this reads easier IMO . at any rate, your call as the result is correct

```python
if not vm or not vm.config:
  return None
return vm.config.name
```

File: esx_service/cli/vmdkops_admin.py [hunk 2]
```diff
@@ -526,7 +526,11 @@ def get_creation_info(metadata):
 def get_attached_to(metadata):
     """ Return which VM a volume is attached to based on its metadata """
     try:
-        return metadata[kv.ATTACHED_VM_NAME]
+        vm_name = vmdk_ops.vm_uuid2name(metadata[kv.ATTACHED_VM_UUID])
+        if not vm_name:
+            return kv.DETACHED
+        else:
```
Comment (msterin) [inline esx_service/cli/vmdkops_admin.py]: nit: 'else' is not needed IMO

**Comment by govint** [comment]
Updated changes to address review comments. Tried running with Python 3.5 and there are exception and print statements that are incompatible. Fixed those also.

**Comment by msterin** [comment]
@govint - the actual fix looks good and ready to merge. The changes related to whitespaces and fixing `except` to comply with 6.5 , while correct, do not belong in the same commit. And generally do not belong in this PR at all. They have a potential to introduce conflicts for PRs still in flight (for tenants), and they do not make product run on the latest ESX (CI will fail there), not do they clean up trailing whitespaces in the whole code base (there are still many)

For now, let's merge (since it's a fairly small set of changes and the commit is squashed) , but please - do  not mix different work in the same commit. Also, I am planning to do a pass on Py3/whitespaces work, so please do not do anymore fixes in this area to avoid conflicts.

**Comment by govint** [comment]
Sure, I couldn't run on 6.5 which I have here as a test bed and hence had
to fix these.

**Comment by kerneltime** [comment]
Fixes https://github.com/vmware/docker-volume-vsphere/issues/738

**Pull request merged**